code/lib/builder-vite/src/plugins/source-loader-plugin.ts

```diff
@@ -11,6 +11,7 @@ const mockClassLoader = (id: string) => ({
   // eslint-disable-next-line no-console
   emitWarning: (message: string) => console.warn(message),
   resourcePath: id,
+  getOptions: () => ({}),
```

Comment:
It looks like the webpack version has a few options to control things like prettier options, the parser used, etc.  I wonder if there's a way we can get a hold of the `sourceLoaderOptions` from the docs addon, which seems to be the way those are typically set.

Reply:
@shilman Any ideas here? I don't think the docs preset currently returns this information